### DISCUSSION TITLE
Why do I see lines or a grid-like pattern on my colour corrected image?

### DISCUSSION THREAD

**Discussion by SamimAB** [colour-science/colour-checker-detection#69] [Q&A]
![compare](https://user-images.githubusercontent.com/82173115/193399049-c7ea8389-7b99-4dfd-b691-24330064ee65.png)
I have used this example (https://github.com/colour-science/colour-checker-detection/blob/master/colour_checker_detection/examples/examples_detection.ipynb) to color correct the above image. But I get these lines after applying color correction, if u zoom these are red , green, blue line. And they rotate if I rotate the original image before processing. I am very new to this any help is appreciated.

**Comment by SamimAB** [comment]
So I found the issue I was loading in RGBA image but the color matrices for swatches RGB , converting the image from RGBA to RGB solved it.

**Reply by KelSolaar** [reply]
Yes, as pointed out on [Stack Overflow](https://stackoverflow.com/questions/73916931/color-correction-using-color-checker) you can also do that: `colour.io.read_image(path)[..., 0:3]`
